src/relay/op/type_relations.cc

```diff
@@ -90,8 +90,8 @@ bool BroadcastRel(const Array<Type>& types,
                   const Attrs& attrs,
                   const TypeReporter& reporter) {
   CHECK_EQ(types.size(), 3);
-  RELAY_LOG(INFO) << "In1: " << types[0] << "In2: " << types[1]
-                  << "Out: " << types[2] << std::endl;
+  RELAY_LOG(INFO) << "In1: " << types[0] << ",In2: " << types[1]
```

Comment:
maybe ", In2:" to keep consistency with https://github.com/dmlc/tvm/blob/master/src/relay/ir/expr.cc?

Reply:
@MarisaKirisame done.